utilities/bash/dot_profile

```diff
@@ -1455,15 +1544,15 @@ function sixdeskRenewKerberosToken(){
     else
 	local __lKlist=true
     fi
-    sixdeskmess -1 " --> kinit;"
+    sixdeskmess 1 " --> kinit;"
```

Comment:
I would keep the -1 level, as this is not a useful echo, but a very important one, to catch if a problematic submission/operation could be due to expired tokens